jmix-flowui/flowui-devserver/flowui-devserver.gradle

```diff
@@ -29,13 +29,11 @@ archivesBaseName = 'jmix-flowui-devserver'
 dependencies {
     api(project(":flowui-kit"))
 
-    api("org.eclipse.jetty:jetty-continuation:$jettyVersion")
     api("org.eclipse.jetty:jetty-webapp:$jettyVersion")
-    api("org.eclipse.jetty.websocket:websocket-server:$jettyVersion")
-    api("org.eclipse.jetty.websocket:javax-websocket-server-impl:$jettyVersion")
-
+    api("org.eclipse.jetty.websocket:websocket-jakarta-server:$jettyVersion")
```

Comment:
Do we need a jettyVersion? Won't it be taken from Spring Boot BOM? The same is for jakarta.servlet:jakarta.servlet-api and commons-logging - will these versions be taken from BOM?